torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1052,6 +1052,19 @@ XLANativeFunctions::convolution_backward_overrideable(
                      : at::Tensor());
 }
 
+at::Tensor XLANativeFunctions::copy(const at::Tensor& self,
+                                    const at::Tensor& src, bool non_blocking) {
+  TORCH_LAZY_FN_COUNTER("xla::");
+  return _copy_from(src, self, non_blocking);
+}
+
+at::Tensor& XLANativeFunctions::copy_(at::Tensor& self, const at::Tensor& src,
```

Comment:
Wait a minute. Yea, I think this additional copy_ lowering should be able to fix the issue I'm facing on the other pull request.

I hit an internal assertion on the other pull request that basically suggest the auto-generated copy_ lowering is wrong.

If the CIs are all green, I will let you handle the copy/copy_ lowering in your PR here. And then I will do a rebase after your merge.